src/Classes/JsonApiData.php

```diff
@@ -4,19 +4,19 @@
 
 namespace App\Classes;
 
-use App\Service\EntityManagerLookup;
+use App\Service\EntityRepositoryLookup;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 
 class JsonApiData
 {
     protected array $data = [];
     protected array $includes = [];
     protected array $sideLoadCandidates = [];
-    protected EntityManagerLookup $entityManagerLookup;
+    protected EntityRepositoryLookup $entityManagerLookup;
     protected NormalizerInterface $normalizer;
 
     public function __construct(
-        EntityManagerLookup $entityManagerLookup,
+        EntityRepositoryLookup $entityManagerLookup,
```

Comment:
I think the variable and property need $entityRepositoryLookup as well.

Reply:
:heavy_check_mark: